### ISSUE TITLE
Transifex upload issue

### ISSUE THREAD

**Issue by DeepDiver1975** [owncloud/ios-sdk#128]
```
# Getting info about resources

owncloud-ios.sdk - Done
owncloud-ios.sdk-ui - Done
[##############################] (2 / 2)

# Pushing source files

owncloud-ios.sdk - failed to upload of resource 'o:owncloud-org:p:owncloud-ios:r:sdk' - parse_error: Multiple different entries for language code: nb_no or NB_NO
owncloud-ios.sdk-ui - failed to upload of resource 'o:owncloud-org:p:owncloud-ios:r:sdk-ui' - parse_error: Multiple different entries for language code: nn_no or NN_NO
[##############################] (2 / 2)
 
```

**Comment by DeepDiver1975** [comment]
@felix-schwarz @hosy any idea? THX

**Comment by DeepDiver1975** [comment]
once this is fixed - please apply this fix to ios-app as well

![image](https://github.com/user-attachments/assets/02985712-7a1f-42f5-9e8b-d28e7b6bb6a9)

**Comment by felix-schwarz** [comment]
Addressed by #130.
